inheritance/src/test/java/inheritance/RestaurantTest.java

```diff
@@ -1,6 +1,29 @@
 package inheritance;
 
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
 public class RestaurantTest {
 
+    @Test
+    public void testRestaurant(){
+        Restaurant roosters = new Restaurant("Roosters", 5, "$$");
+        assertEquals(roosters.getName(), "Roosters");
+        assertEquals(roosters.getStars(), 5);
+        assertEquals(roosters.getPrice(), "$$");
+    }
+
+    @Test public void toStringTest(){
+        assertTrue(String.format("Roosters", "$$", "Minnie Mouse", 4, "Best wings ever!"), true);

```

Comment:
this test is asserting that true is true, it should assert equals (roosters.toString(), "Roosters is a restaurant etc...")